### PR TITLE
Documenting how to modify individual policies

### PR DESCRIPTION
**Pull request by Brunoga-MS**
# Overview/Summary

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## This PR fixes/adds/changes/removes
- Documenting individual policy change process
- Aligning documentation for multiple email addresses
- Fixing typos on:
  - _index.md
  - Customize-Policy-Assignment.md related to [[Question/Feedback]: Modifying policy definitions clarification #24](https://github.com/Azure/azure-monitor-baseline-alerts/issues/24) and workitem [GH#24 AMBA: Clarification for modifying individual policies](https://dev.azure.com/CSUSolEng/Azure%20Landing%20Zones/_workitems/edit/31242)
  - Introduction-to-deploying-the-ALZ-pattern.md
- Fixing ALZMonitorResourceGroupTags value in the parameter file
- Removing the customization of ALZMonitorResourceGroupTags

### Breaking Changes

1. NONE

## As part of this Pull Request I have

- [X] Read the Contribution Guide and ensured this PR is compliant with the guide
- [X] Checked for duplicate [Pull Requests](https://github.com/Azure/azure-monitor-baseline-alerts/pulls)
- [X] Associated it with relevant [GitHub Issues](https://github.com/Azure/azure-monitor-baseline-alerts/issues) or ADO Work Items (Internal Only)
- [X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/azure-monitor-baseline-alerts/)
- [X] Ensured PR tests are passing
- [X] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: docs/content/patterns/alz/deploy/Customize-Policy-Assignment.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ The parameter file contains the same default values as listed in our documentati
         "policyAssignmentParametersCommon": {
             "value": {
                 "ALZMonitorResourceGroupName": {
-                    "value": "rg-alz-monitor"
+                    "value": "rg-amba-monitoring-001"
                 },
                 "ALZMonitorResourceGroupTags": {
                     "value": {
-                        "Project": "alz-monitor"
+                        "_deployed_by_amba": true
```
Comment (arjenhuitema) [inline docs/content/patterns/alz/deploy/Customize-Policy-Assignment.md:42]: We should keep "Project": "amba-monitoring" as the example value. This is for customers to change should they wish to add tags to the RG.

File: docs/content/patterns/alz/deploy/Customize-Policy-Assignment.md [hunk 2]
```diff
@@ -82,7 +82,6 @@ The following parameters can be changed for activity log, service health alert a
 | **Parameter Name** | **Parameter Description** |
 |----------|----------|
 | ALZMonitorResourceGroupName | The name of the resource group to place the alerts in |
-| ALZMonitorResourceGroupTags | Any tags than needs to be added to the resource group created |
```
Comment (arjenhuitema) [inline docs/content/patterns/alz/deploy/Customize-Policy-Assignment.md:85]: We should keep this.

File: docs/content/patterns/alz/deploy/Deploy-with-Azure-CLI.md [hunk 3]
```diff
@@ -17,11 +17,17 @@ The following changes apply to all scenarios, whether you are aligned or unalign
 
 - Change the value of _enterpriseScaleCompanyPrefix_ to the management group where you wish to deploy the policies and the initiatives. This is usually the so called "pseudo root management group", e.g. in [ALZ terminology](https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups), this would be the so called "Intermediate Root Management Group" (directly beneath the "Tenant Root Group").
 - Change the value of _ALZMonitorResourceGroupName_ to the name of the resource group where the activity logs, resource health alerts, actions groups and alert processing rules will be deployed in.
-<!--
-- Change the value of _ALZMonitorResourceGroupTags_ to specify the tags to be added to said resource group.
```
Comment (arjenhuitema) [inline docs/content/patterns/alz/deploy/Deploy-with-Azure-CLI.md:21]: We should keep this also.

File: docs/content/patterns/alz/deploy/Deploy-with-Azure-CLI.md [hunk 4]
```diff
@@ -105,7 +111,7 @@ Note that the parameter file shown below has been truncated for brevity, compare
                 },
                 "ALZMonitorResourceGroupTags": {
                     "value": {
-                        "Project": "amba-monitoring"
+                        "_deployed_by_amba": true
```
Comment (arjenhuitema) [inline docs/content/patterns/alz/deploy/Deploy-with-Azure-CLI.md:114]: We should keep the original example value for the tag.

File: docs/content/patterns/alz/deploy/Deploy-with-Azure-PowerShell.md [hunk 5]
```diff
@@ -17,11 +17,17 @@ The following changes apply to all scenarios, whether you are aligned or unalign
 
 - Change the value of _enterpriseScaleCompanyPrefix_ to the management group where you wish to deploy the policies and the initiatives. This is usually the so called "pseudo root management group", e.g. in [ALZ terminology](https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups), this would be the so called "Intermediate Root Management Group" (directly beneath the "Tenant Root Group").
 - Change the value of _ALZMonitorResourceGroupName_ to the name of the resource group where the activity logs, resource health alerts, actions groups and alert processing rules will be deployed in.
-<!--
-- Change the value of _ALZMonitorResourceGroupTags_ to specify the tags to be added to said resource group.
```
Comment (arjenhuitema) [inline docs/content/patterns/alz/deploy/Deploy-with-Azure-PowerShell.md:21]: We should keep this

File: docs/content/patterns/alz/deploy/Deploy-with-Azure-PowerShell.md [hunk 6]
```diff
@@ -105,7 +111,7 @@ Note that the parameter file shown below has been truncated for brevity, compare
                 },
                 "ALZMonitorResourceGroupTags": {
                     "value": {
-                        "Project": "amba-monitoring"
+                        "_deployed_by_amba": true
```
Comment (arjenhuitema) [inline docs/content/patterns/alz/deploy/Deploy-with-Azure-PowerShell.md:114]: We should keep the original example value for the tag.

File: docs/content/patterns/alz/deploy/Deploy-with-GitHub-Actions.md [hunk 7]
```diff
@@ -17,11 +17,17 @@ The following changes apply to all scenarios, whether you are aligned or unalign
 
 - Change the value of _enterpriseScaleCompanyPrefix_ to the management group where you wish to deploy the policies and the initiatives. This is usually the so called "pseudo root management group", e.g. in [ALZ terminology](https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups), this would be the so called "Intermediate Root Management Group" (directly beneath the "Tenant Root Group").
 - Change the value of _ALZMonitorResourceGroupName_ to the name of the resource group where the activity logs, resource health alerts, actions groups and alert processing rules will be deployed in.
-<!--
-- Change the value of _ALZMonitorResourceGroupTags_ to specify the tags to be added to said resource group.
```
Comment (arjenhuitema) [inline docs/content/patterns/alz/deploy/Deploy-with-GitHub-Actions.md:21]: We should keep this.

File: docs/content/patterns/alz/deploy/Deploy-with-GitHub-Actions.md [hunk 8]
```diff
@@ -105,7 +111,7 @@ Note that the parameter file shown below has been truncated for brevity, compare
                 },
                 "ALZMonitorResourceGroupTags": {
                     "value": {
-                        "Project": "amba-monitoring"
+                        "_deployed_by_amba": true
```
Comment (arjenhuitema) [inline docs/content/patterns/alz/deploy/Deploy-with-GitHub-Actions.md:114]: We should keep the original example value for the tag.

File: patterns/alz/alzArm.param.json [hunk 9]
```diff
@@ -45,7 +45,7 @@
         },
         "ALZMonitorResourceGroupTags": {
           "value": {
-            "Project": "amba-monitoring"
+            "_deployed_by_amba": true
```
Comment (arjenhuitema) [inline patterns/alz/alzArm.param.json:48]: We should keep the original example value for the tag.

**Review by arjenhuitema — CHANGES_REQUESTED** [review]
Great work Bruno. Let's park the tagging discussion untill Friday and get the other items in with this PR.

File: docs/content/patterns/alz/deploy/Deploy-with-Azure-PowerShell.md (hunk 5 above)
Comment (Brunoga-MS) [reply docs/content/patterns/alz/deploy/Deploy-with-Azure-PowerShell.md:21]: Done

File: docs/content/patterns/alz/deploy/Deploy-with-Azure-PowerShell.md (hunk 6 above)
Comment (Brunoga-MS) [reply docs/content/patterns/alz/deploy/Deploy-with-Azure-PowerShell.md]: Done

File: docs/content/patterns/alz/deploy/Customize-Policy-Assignment.md (hunk 2 above)
Comment (Brunoga-MS) [reply docs/content/patterns/alz/deploy/Customize-Policy-Assignment.md]: Done

File: docs/content/patterns/alz/deploy/Deploy-with-Azure-CLI.md (hunk 3 above)
Comment (Brunoga-MS) [reply docs/content/patterns/alz/deploy/Deploy-with-Azure-CLI.md:21]: Done

File: docs/content/patterns/alz/deploy/Deploy-with-Azure-CLI.md (hunk 4 above)
Comment (Brunoga-MS) [reply docs/content/patterns/alz/deploy/Deploy-with-Azure-CLI.md]: Done

File: docs/content/patterns/alz/deploy/Deploy-with-Azure-Pipelines.md [hunk 10]
```diff
@@ -105,7 +111,7 @@ Note that the parameter file shown below has been truncated for brevity, compare
                 },
                 "ALZMonitorResourceGroupTags": {
                     "value": {
-                        "Project": "amba-monitoring"
+                        "_deployed_by_amba": true
```
[the remark this answers is not in the document]
Comment (Brunoga-MS) [reply docs/content/patterns/alz/deploy/Deploy-with-Azure-Pipelines.md]: Done

**Review by Brunoga-MS — COMMENTED** [review]
All comments addressed

**Review by arjenhuitema — APPROVED** [review]
LGTM

**Pull request merged**